docs/ref/forms/api.txt

```diff
@@ -260,9 +260,16 @@ precedence::
 
 .. method:: Form.get_initial_for_field(field, field_name)
 
-Use :meth:`~Form.get_initial_for_field()` to retrieve initial data for a form
-field. It retrieves data from :attr:`Form.initial` and :attr:`Field.initial`,
-in that order, and evaluates any callable initial values.
+Returns the initial data for a form field. The value is retrieved from
+:attr:`Form.initial`, or :attr:`Field.initial` if not present, and then
+evaluated if a callable.
+
+It is recommended to use :attr:`BoundField.initial` over
+:meth:`~Form.get_initial_for_field()` because ``BoundField.initial`` has a
+simpler interface. Also, unlike :meth:`~Form.get_initial_for_field()`,
+:attr:`BoundField.initial` caches its values. This is useful especially when
```

Comment:
Maybe 'especially useful'. Appreciate this is marginal! 🙂

Reply:
After thinking more, this changes the meaning from what I was intending, IMO. Putting it before makes it read more like "very useful," but I was after especially's other meaning of "in particular." (It's not that it's _very_ useful, but rather that the usefulness is apparent more in these callable cases.)